plugin/clang.vim

```diff
@@ -840,10 +840,27 @@ func! s:ClangCompleteInit(force)
 endf
 "}}}
 "{{{ ClangExecuteNeoJobHandler
-"handles event: exit
+"Handles events: stdout, stderr, exit
 func! ClangExecuteNeoJobHandler(job_id, data, event)
-  if a:event == 'exit'
-    call ClangExecuteDone(self.fstdout, self.fstderr)
+  if a:event == 'stdout' || a:event == 'stderr'
+    if len(b:clang_state[a:event]) != 0
+      if b:clang_state[a:event][-1] == ''
+        call remove(b:clang_state[a:event], -1)
+      else
+        let line = a:data[0]
+        let b:clang_state[a:event][-1] .= line
+        call remove(a:data, 0)
+      endif
+    endif
```

Comment:
I found the result is strange, `a:data` is not separated line by line, may appear something like:
```
COMPLETION: symlinkat : [#int#]symlinkat(<#const char *__from#>, <#int __tofd#>, <#const char *__to#>)
COMPLETION: sync : [#void#]sync()
COMPLETION: sys_errlist : [#const char *const []#]sys_errlist
COMPLETIO
N: sys_nerr : [#int#]sys_nerr
COMPLETION: syscall : [#long#]syscall(<#long __sysno, ...#>)
COMPLETION: sysconf : [#long#]sysconf(<#int __name#>)
```

So `a:data` is 
```vim
['xxx', 'xxxx', '']
```

for `stdout` and `stderr`